### DISCUSSION TITLE
Supress validation for a line/file

### DISCUSSION THREAD

**Discussion by jasperfirecai2** [github/vscode-github-actions#59] [Q&A] [answered]
IS there any way to supress a specific warning for a file or line? A bunch of my workflows have a warning that Context access **might** be invalid because it's a dynamic value, but i know it is never the case. I just want the warning gone.

**Comment by KetchupOnMyKetchup** [comment, marked as answer]
There isn't currently support for this, but we think this is a super interesting feature.  Could you please file an enhancement issue for this and we can take a look at prioritizing it.

**Reply by inean** [reply]
This thread is a bit old. Was the  enhancement request filled ?

**Reply by jasperfirecai2** [reply]
https://github.com/github/vscode-github-actions/issues/123 issue still open
